templates/environment/versions/cf-v1.5.0.yml

```diff
@@ -0,0 +1,404 @@
+{{- if .ImportVPC}}
+      VpcId: {{.ImportVPC.ID}}
+{{- else}}
+      VpcId: !Ref VPC
+{{- end}}
+      Tags:
+        - Key: Name
+          Value: !Sub 'copilot-${AppName}-${EnvironmentName}-efs'
+  EFSSecurityGroupIngressFromEnvironment:
+    Type: AWS::EC2::SecurityGroupIngress
+    Condition: CreateEFS
+    Properties:
+      Description: Ingress from containers in the Environment Security Group.
+      GroupId: !Ref EFSSecurityGroup
+      IpProtocol: -1
+      SourceSecurityGroupId: !Ref EnvironmentSecurityGroup
+{{- if .ImportVPC}}
+{{- range $ind, $id := .ImportVPC.PrivateSubnetIDs}}
+  MountTarget{{inc $ind}}:
+    Type: AWS::EFS::MountTarget
+    Condition: CreateEFS
+    Properties:
+      FileSystemId: !Ref FileSystem
+      SubnetId: {{$id}}
+      SecurityGroups:
+        - !Ref EFSSecurityGroup
+{{- end}}
+{{- else}}
+{{- range $ind, $cidr := .VPCConfig.PrivateSubnetCIDRs}}
+  MountTarget{{inc $ind}}:
+    Type: AWS::EFS::MountTarget
+    Condition: CreateEFS
+    Properties:
+      FileSystemId: !Ref FileSystem
+      SubnetId: !Ref PrivateSubnet{{inc $ind}}
+      SecurityGroups:
+        - !Ref EFSSecurityGroup
+{{- end}}
+{{- end}}
+{{include "cfn-execution-role" . | indent 2}}
+{{include "environment-manager-role" . | indent 2}}
+{{include "custom-resources-role" . | indent 2}}
+  EnvironmentHostedZone:
+    Type: "AWS::Route53::HostedZone"
+    Condition: DelegateDNS
+    Properties:
+      HostedZoneConfig:
+        Comment: !Sub "HostedZone for environment ${EnvironmentName} - ${EnvironmentName}.${AppName}.${AppDNSName}"
+      Name: !Sub ${EnvironmentName}.${AppName}.${AppDNSName}
+{{include "lambdas" . | indent 2}}
+{{include "custom-resources" . | indent 2}}
+Outputs:
+  VpcId:
+{{- if .ImportVPC}}
+    Value: {{.ImportVPC.ID}}
+{{- else}}
+    Value: !Ref VPC
+{{- end}}
+    Export:
+      Name: !Sub ${AWS::StackName}-VpcId
+  PublicSubnets:
+{{- if .ImportVPC}}
+    Value: !Join [ ',', [ {{range $id := .ImportVPC.PublicSubnetIDs}}{{$id}}, {{end}}] ]
+{{- else}}
+    Value: !Join [ ',', [ {{range $ind, $cidr := .VPCConfig.PublicSubnetCIDRs}}!Ref PublicSubnet{{inc $ind}}, {{end}}] ]
+{{- end}}
+    Export:
+      Name: !Sub ${AWS::StackName}-PublicSubnets
+  PrivateSubnets:
+{{- if .ImportVPC}}
+    Value: !Join [ ',', [ {{range $id := .ImportVPC.PrivateSubnetIDs}}{{$id}}, {{end}}] ]
+{{- else}}
+    Value: !Join [ ',', [ {{range $ind, $cidr := .VPCConfig.PrivateSubnetCIDRs}}!Ref PrivateSubnet{{inc $ind}}, {{end}}] ]
+{{- end}}
+    Export:
+      Name: !Sub ${AWS::StackName}-PrivateSubnets
+  ServiceDiscoveryNamespaceID:
+    Condition: UseLegacyServiceDiscovery
+    Value: !GetAtt ServiceDiscoveryNamespace.Id
+    Export:
+      Name: !Sub ${AWS::StackName}-ServiceDiscoveryNamespaceID
+  ServiceDiscoveryNamespaceIDWithEnv:
+    Value: !GetAtt ServiceDiscoveryNamespaceWithEnv.Id
+    Export:
+      Name: !Sub ${AWS::StackName}-ServiceDiscoveryNamespaceIDWithEnv
+  UseLegacyServiceDiscovery:
```

Comment:
Why do we need this output? Can't we check if `UseLegacyServiceDiscoveryIfBlank` this param is blank to tell?